infrastructure/template/root/setup.cfg

```diff
@@ -0,0 +1,6 @@
+[flake8]
+# Black compatible values https://black.readthedocs.io/en/stable/compatible_configs.html#flake8
+max-line-length = 88
+extend-ignore =
+            E203, # whitespace before ':'
+            W503  # line break before binary operator
```

Comment:
W503 is ignored by default by flake8. So no need to specifically ignore.
https://github.com/psf/black/pull/1661

Reply:
Good catch! I will remove that.